modules/cli-logging-in.adoc

```diff
@@ -51,13 +58,27 @@ You don't have any projects. You can try to create a new project, by running
 
 Welcome! See 'oc help' to get started.
 ----
-<1> Enter the {product-title} server URL.
+<1> Enter the 
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+server URL.
 <2> Enter whether to use insecure connections.
 <3> Enter the user's password.
 
 [NOTE]
 ====
-If you are logged in to the web console, you can generate an `oc login` command that includes your token and server information. You can use the command to log in to the {product-title} CLI without the interactive prompts. To generate the command, select *Copy login command* from the username drop-down menu at the top right of the web console.
+If you are logged in to the web console, you can generate an `oc login` command that includes your token and server information. You can use the command to log in to the 
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
```

Comment:
```suggestion
OpenShift
```
I think we're still using `oc` here, not `rosa`. Wonder if this should be "OpenShift?"